charts/latest/spdk-csi/templates/setup-snapshot-controller.yaml

```
@@ -0,0 +1,43 @@
+# This YAML file shows how to deploy the snapshot controller
+
+# The snapshot controller implements the control loop for CSI snapshot functionality.
+# It should be installed as part of the base Kubernetes distribution in an appropriate
+# namespace for components implementing base system functionality. For installing with
+# Vanilla Kubernetes, kube-system makes sense for the namespace.
+{{- if .Values.snapshotclass.create -}}
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: snapshot-controller
+  namespace: kube-system
```

```suggestion
 namespace: {{ .Release.Namespace }}
```

please make sure that the all the controllers are installed in the same namespace

---

snapshot-controller needs to be installed in the `kube-system` namespace